pkg/disk/disk_test.go

```diff
@@ -164,6 +164,7 @@ func TestCreatePartitionTable(t *testing.T) {
 			// /boot and subdirectories is exempt from this rule
 			return nil
 		}
+
```

Comment:
(nitpick) This change in the ` disk: add a test partition table with swap ` looks a bit unrelated(?)

Reply:
I disagree.  It's definitely completely unrelated.